backend/main.py

```diff
@@ -1,6 +1,32 @@
-from fastapi import FastAPI
+from fastapi import FastAPI, status, HTTPException
+from pydantic import BaseModel
+import bcrypt
+
+#User object to save user info
+class User(BaseModel):
+    email: str
+    password: str
+    username: str
```

Comment:
should be in schema folder 
schemas are for structure of data that the API wants and returns

Reply:
lets put this in models folder instead. Lets reserve schemas for db sqlalchemy things if needed in future